src/types/acl.graphql

```diff
@@ -31,8 +31,71 @@ type Acl {
 
   "Full JSON of the Acl."
   acl: JSON @deprecated(reason: "Deprecating in favor of `groupPermissions`, `legacyGuid`, `catalogItemIdentity`.")
+
+  collections (
+    "Collections query parameters"
+    params: CollectionsInput
+  ) : CollectionList
+
+}
+
+type CatalogItemIdentity {
+  "Is collection applicable for a ACL"
+  collectionApplicable: Boolean
+  
+  "The collection information"
+  collectionIdentifier: JSON
+
+  "Is granule applicable for a ACL"
+  granuleApplicable: Boolean
+  
+  "The granule information"
+  granuleIdentifier: JSON
 }
 
+type AclGroupList {
+  "The number of hits for a given search."
+  count: Int
+
+  "The list of group search results."
+  items: [AclGroup]
+}
+
+type AclGroup {
+  "The unique id assigned to the group."
+  groupId: String
```

Comment:
In the Group type we renamed this field to `id`, @abbottry do you want to do that here too?

Reply:
You are correct, Ryan did mention to update that but I forgot. Updated and pushed